### ISSUE TITLE
Hypertune parameters

### ISSUE THREAD

**Issue by malteserteresa** [opt-out-tools/study-online-misogyny#8]
**Objective**
We currently have four baseline deep learning models that require hyperparameter tuning.

**Description**
Hyperparameter tuning can be done by guess work (not recommended), grid search, random search or bayesian optimization (using libraries such as hyperas or hyperopt). 

**Skills**

**Tools**
hyperas/hyperopt

**Time Estimation**

**Detailed description/notes**
Parameters to tune
- Tokenizer - num_words
- tokenizer - modes (e.g., tf-idf)
- Number of layers
- Dimensionality of layers
- Number of epochs
- Dropout
- L1 / L2 reg
- Learning rate optimizer
